dump/dump.go

```diff
@@ -0,0 +1,51 @@
+package dump
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/gunk/gunk/generate"
+)
+
+// Run will generate the FileDescriptorSet for a Gunk package, and
+// output it as required.
+func Run(fileToWrite, format, dir string, patterns ...string) error {
```

Comment:
I don't know how easy it is to do that on Windows? Happy to remove it if it is trivial on Windows?

Reply:
It does seem like windows CMD supports the basic posix-y redirects: https://ss64.com/nt/syntax-redirection.html

Reply:
Ah sweet, I was under the impression it was a bit more difficult which was why most commands have this functionality built in. Removed it.

Reply:
I think the reason tends to be so that it's trivial to dump to a file when one doesn't have a shell, e.g. `exec.Command("gunk", "dump", "-o", myfile)`. But I argue that instead, the user should the same thing that a shell would do, i.e. `Command("gunk", "dump")` and then pipe that to a file directly.